### ISSUE TITLE
Payolution postponed capture

### ISSUE THREAD

**Issue by JEreth** [PAYONE-GmbH/magento-1#146]
The payment methods of Payolution requiere to set an online capture only after shipment. To automate this within Magento, one can listen to the _sales_order_shipment_save_after_ event and then capture the open amount.

We created a simple extension that adds that functionality:
https://github.com/Pragmatic-Apps/magento1-payone-postponecapture

Maybe this could also be a feature for a future version of this extension, as the manual capture processes is hard when the number of orders increase.

**Comment by fjbender** [comment]
Thanks! This will come in handy for some of our merchants I'm sure. I'll take a look and see if we add this as a feature directly into our module (with credits where credits are due, of course).

**Comment by fjbender** [comment]
@JEreth It'll be a big break in expected module behavior if we ship this out in the standard package. With a6e4ea4ca7f6eca2c6a1b273058001821221537c, however, I added a "Related projects" section where your extension is featured. Hope this is fine with you, otherwise I'll revert a6e4ea4ca7f6eca2c6a1b273058001821221537c.
